shared/src/main/kotlin/org/javacs/kt/classpath/MavenClassPathResolver.kt

```diff
@@ -125,8 +126,14 @@ private fun runCommand(pom: Path, command: String) {
     }
 }
 
-private val mvnCommand: Path by lazy {
-    requireNotNull(findCommandOnPath("mvn")) { "Unable to find the 'mvn' command" }
+private val mvnCommandFromPath: Path? by lazy {
+    findCommandOnPath("mvn")
+}
+
+private fun mvnCommand(pom : Path) : Path {
+	return requireNotNull(mvnCommandFromPath ?: findProjectCommandWithName("mvnw", pom)?.also {
+		LOG.info("Using mvn wrapper (mvnw) in place of mvn command")
+	}) { "Unable to find the 'mvn' command or suitable wrapper" }
 }
```

Comment:
Please indent consistently with 4 spaces (I recommend adding the EditorConfig extension to your editor/IDE to care of this automatically) and format type annotations as `x: T` rather than `x : T`.